### PR TITLE
Add Civilians page

### PR DESCRIPTION
**Pull request by aileendds**
## Checklist

I have…

- [x] run the application locally (`bin/rails server`) and verified that my changes behave as expected.
- [x] run static code analysis (`bin/rubocop`) and vulnerability scan (`bin/brakeman`) against my changes.
- [x] run the test suite (`bin/rake spec`) and verified that all tests pass.
- [x] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [x] thoroughly outlined below the steps necessary to test my changes.
- [x] attached screenshots illustrating relevant behavior before and after my changes.

## Summary of Changes

This pull request adds the civilians page in zeplin. Since a lot of the content is the same as OCONUS, I also made some of the same changes there that I made to civilians.

Issue #179

## Testing

To verify the changes proposed in this pull request…

clone this repo,
git checkout civilians,
set up development dependencies according to CONTRIBUTING.md,
run bin/rails server, and
load up http://localhost:3000 in your Web browser of choice.

## Screenshots

<details><summary>Desktop</summary>

screencapture-github-deptofdefense-move-mil-issues-179-1512761102037

</details>

<details><summary>Mobile</summary>

screencapture-localhost-3000-moving-guide-civilians-1512761115222

</details>

### CODE REVIEW

File: app/views/pages/moving-guide/civilians.html.erb [hunk 1]
```diff
@@ -1,11 +1,331 @@
...
+
+      <p>Sometimes, your command may authorize a <strong>House Hunting Trip (HHT)</strong>. This discretionary allowance will reimburse you for meals and lodging for up to 10 days, while you look for a new place to live in your new city. Check out the <%= link_to 'Reimbursements & Vouchers', page_path('moving-guide/reimbursements') %> page for details.</p>
+    </div>
+
+    <div id="orders" class="article-section">
+      <h4>Orders</h4>
+
+      <p>
+        <strong><em>Accuracy Matters</em></strong><br>
+        When you receive your orders, make sure all the details are accurate. In addition to the administrative details (your name, social security number, etc.), make sure it has the correct duty station, dependent information, and reporting dates. Let the administration section know that there is an error so it can be corrected as soon as possible.
```
Comment (jamesatheyDDS) [inline app/views/pages/moving-guide/civilians.html.erb]: ~that~ **if** there is an error

File: app/views/pages/moving-guide/civilians.html.erb [hunk 2]
```diff
@@ -1,11 +1,331 @@
...
+        <strong>Pro-Tip:</strong> Get a General Power of Attorney for your spouse. This allows them to take necessary action on moving and financial decisions while you're at work.
+      <% end %>
+
+      <p>Once you have your orders, you can log in to the <%= link_to 'Defense Personal Property System', 'https://eta.sddc.army.mil/ETASSOPortal/default.aspx' %> to schedule your move.</p>
+    </div>
+
+    <div id="scheduling" class="article-section">
+      <h4>Schedule Your Move</h4>
+
+      <p>1.) Use the <%= link_to 'Defense Personal Property System', 'https://eta.sddc.army.mil/ETASSOPortal/default.aspx' %> to upload your orders and create a shipment (or two). Check out these <%= link_to 'tutorials', tutorials_path %> for help using <%= abbr_tag('dps') %>.</p>
```
Comment (jamesatheyDDS) [inline app/views/pages/moving-guide/civilians.html.erb]: Why not &lt;ol&gt; and &lt;li&gt; (ordered list) instead?

File: app/views/pages/moving-guide/civilians.html.erb [hunk 3]
```diff
@@ -1,11 +1,331 @@
...
+      <p>2.) Next, take the DD forms generated by <%= abbr_tag('dps') %> to your local <%= link_to 'Transportation Office', offices_path %>. At that time, you'll get matched with a moving company (also known as a Transportation Service Provider, or Carrier), and your packing and moving dates will be confirmed.</p>
+
+      <div class="usa-grid-full">
+        <div class="usa-width-one-third">
+          Have this info ready for <%= abbr_tag('dps') %>:
+          <ul>
+            <li>Your contact information</li>
+            <li><%= link_to 'Estimated weight', weight_estimator_path %></li>
+            <li>Pick up &amp; delivery locations</li>
+            <li>Pick up & delivery dates</li>
```
Comment (jamesatheyDDS) [inline app/views/pages/moving-guide/civilians.html.erb]: replace &amp; with &amp;amp;

File: app/views/pages/moving-guide/civilians.html.erb [hunk 4]
```diff
@@ -1,11 +1,331 @@
...
+          <li>Choose wisely. A household goods shipment won't arrive at your new location for several weeks.</li>
+          <li>When you pack for an overseas move, keep an accordian folder in your carry on bag containing copies of important documents you might need when going through customs in your new host country.</li>
+        </ul>
+      <% end %>
+    </div>
+
+    <div id="pack-day" class="article-section">
+      <h4>Packing Day</h4>
+
+      <p>Your packing day and moving dates are not the same thing. On packing day, the moving company sends a team of packers to your home and puts all your items into boxes, usually as fast as possible. The day your boxed are loaded onto the truck for moving day could be several days later.</p>
```
Comment (jamesatheyDDS) [inline app/views/pages/moving-guide/civilians.html.erb]: The day your ~boxed~ **boxes**

File: app/views/pages/moving-guide/civilians.html.erb [hunk 5]
```diff
@@ -1,11 +1,331 @@
...
+              <li><strong>Physically separate</strong> anything you plan to take/keep with you from the rest of your belongings into a closet or your vehicle before the packers arrive. Clearly mark this area with a "Do Not Pack" sign.</li>
+              <li><strong>Invite friends and family</strong> to keep an eye on things during packing day. It can be really helpful if you can have someone shadowing the packers, making sure your inventory is accurate, potentially even taking photos of the inside of boxes as they are packed.</li>
+              <li><strong>Label your boxes</strong> with a small piece of duct tape, each room with its own color. Others have found it helpful to number each room when you arrive in your host country, so that your movers know that the master bedroom is 1, the kitchen 2, and the living room 3.</li>
+              <li><%= link_to 'View more tips', page_path('moving-guide/tips') %></li>
+            </ul>
+          <% end %>
+        </div>
+      </div>
+      <p>Bottom line, if at any point something doesn't feel right, you should call your local personal property office and ask for the assistance from an inspector/quality assurance personnel.</p>
```
Comment (jamesatheyDDS) [inline app/views/pages/moving-guide/civilians.html.erb]: There are links to the office locator every other time this text appears; should it go here too?

File: app/views/pages/moving-guide/civilians.html.erb [hunk 6]
```diff
@@ -1,11 +1,331 @@
...
+
+      <%= render layout: 'shared/pro_tip' do %>
+        <strong>Pro-Tip:</strong> If you're traveling with pets, you can check out pet-friendly hotels on the Web at <%= link_to 'www.petswelcome.com', 'https://www.petswelcome.com/' %>.
+      <% end %>
+
+      <p>If you’re moving <%= abbr_tag('oconus') %>, make sure everyone in your family has a valid passport unless you’re headed to Hawaii or Alaska. Family members not listed on the Orders may need visas. When you arrive, expect a few in-processing steps. Depending on your host country, this could include briefings at different offices, and trips to different bases, immediately upon arrival. <%= link_to 'Transportation Office Locator', offices_path %>
+
+      <p>During your travel keep any receipts of your out of pocket expenses,  things like hotel receipts, rental car, gasoline, tolls, etc. that may be needed when you file your voucher at the end. Visit the <%= link_to 'Reimbursements & Vouchers', page_path('moving-guide/reimbursements') %> page for details.</p>
```
Comment (jamesatheyDDS) [inline app/views/pages/moving-guide/civilians.html.erb]: A few small suggestions:

1. bold the "keep any receipts"
1. maybe say "any and all receipts" or just "all receipts"
1. remove the double-space after the first comma

File: app/views/pages/moving-guide/civilians.html.erb [hunk 7]
```diff
@@ -1,11 +1,331 @@
...
+        Keep the lines of communication open and available for the driver, for <%= abbr_tag('conus') %> moves. If your driver has any updates, they may try to contact you as they make their way to your next destination and it’s important you are available to answer any questions that they may have.
+      </p>
+
+      <p>If you’re doing a door-to-door, it’s important that you arrive before the driver. If the driver arrives at your new residence before you do, they may have to put your items in storage, depending on their schedule. For those whose items are going into storage, you have more flexibility with your timing.</p>
+    </div>
+
+    <div id="pets" class="article-section">
+      <h4>Traveling with Pets</h4>
+
+      <p>First, research your new destination’s rules and regulations about bringing pets. Start by contacting your <%= link_to 'local transportation office', offices_path %> to see if there are any breed restrictions, quarantine requirements, and timeline for vaccinations. Check the <%= link_to 'Defense Travel’s Pet site', 'http://www.defensetravel.dod.mil/site/faqpet.cfm' %> for more information.</p>
```
Comment (jamesatheyDDS) [inline app/views/pages/moving-guide/civilians.html.erb]: "Check the Defense Travel’s Pet site for more information" sounds awkward to my ears. Perhaps "Check the DoD Center for Travel Excellence's Pets FAQ for more information?"

File: app/views/pages/moving-guide/civilians.html.erb [hunk 8]
```diff
@@ -1,11 +1,331 @@
...
+          <%= render layout: 'shared/pro_tip' do %>
+            <strong>Pro-Tip:</strong> Enlist your family or your Sponsor for help during this during this phase.
+          <% end %>
+        </div>
+      </div>
+
+      <p>Finally, you have a couple of options during the delivery phase about what items you want the delivery crew to unpack.</p>
+
+      <ul>
+	      <li>A full unpack: Movers empty the boxes of all their contents into the specified rooms.</li>
```
Comment (jamesatheyDDS) [inline app/views/pages/moving-guide/civilians.html.erb]: Minor quibble: indentation here is off.

File: app/views/pages/moving-guide/civilians.html.erb [hunk 9]
```diff
@@ -1,11 +1,331 @@
...
+        Once you have emptied out all your household goods from the boxes, call the local moving company and set up a time for them to come back out and remove any old boxes, packing material, etc. They are responsible for removing it from the residence one time after delivery.
+      </p>
+
+      <%= render layout: 'shared/pro_tip' do %>
+        <strong>Pro-Tip:</strong> Save time by piling up all the boxes and moving supplies in one location!
+      <% end %>
+
+      <p>
+        <strong><em>Claims</em></strong><br>
+        While we hope all of your items arrived in one piece, the reality is that isn’t always the case. Sign in to <%= link_to 'Defense Personal Property System', 'https://eta.sddc.army.mil/ETASSOPortal/default.aspx' %> and complete this part of the process. When you log in to the system, you can expect to see two paths - Lost/Damage Report and Claims Submission.
```
Comment (jamesatheyDDS) [inline app/views/pages/moving-guide/civilians.html.erb]: Either "Sign into DPS" or "Sign into **the** Defense Personal Property System"

File: app/views/pages/moving-guide/civilians.html.erb [hunk 10]
```diff
@@ -1,11 +1,331 @@
...
+        <strong><em>Clean-Up</em></strong><br>
+        Once you have emptied out all your household goods from the boxes, call the local moving company and set up a time for them to come back out and remove any old boxes, packing material, etc. They are responsible for removing it from the residence one time after delivery.
+      </p>
+
+      <%= render layout: 'shared/pro_tip' do %>
+        <strong>Pro-Tip:</strong> Save time by piling up all the boxes and moving supplies in one location!
+      <% end %>
+
+      <p>
+        <strong><em>Claims</em></strong><br>
```
Comment (jamesatheyDDS) [inline app/views/pages/moving-guide/civilians.html.erb]: Is the content in this section identical to that on other pages? If so, should it be a partial?

File: app/views/pages/moving-guide/civilians.html.erb [hunk 11]
```diff
@@ -1,11 +1,331 @@
...
+
+      <p>If you did not submit a Loss/Damage Report, you will need to file your claim within 75 days of delivery. If you submitted a Loss/Damage Report within the 75 day window, you will have up to 9 months to file a claim for the Full Replacement Value of the item you are claiming. If the claim is filed more than 9 months from the delivery date, you will only be eligible for depreciated value up to two years from delivery.</p>
+
+      <p>
+        <strong><em>Customer Satisfaction Survey</em></strong><br>
+        The survey asks some questions about your experience is read by actual humans! Be upfront and honest in your feedback, because if there is a negative trend on a moving company they can be suspended temporarily or indefinitely if they are providing poor service. On the flip side, if you find a moving company that did an excellent job, please put that in the survey as well to help out your colleagues!
```
Comment (jamesatheyDDS) [inline app/views/pages/moving-guide/civilians.html.erb]: Either "The survey **that** asks some questions about your experience is read by actual humans!" or "The survey asks some questions about experience**, and** is read by actual humans!"

Comment (jamesatheyDDS) [reply app/views/pages/moving-guide/civilians.html.erb]: "If there is a negative trend on a moving company _comma_"

Comment (jamesatheyDDS) [reply app/views/pages/moving-guide/civilians.html.erb]: There's another _if_ clause at the end of that sentence. There should be only the if clause at the begin or at the end, not both.

File: app/views/pages/moving-guide/civilians.html.erb (hunk 7 above)
Comment (aileendds) [reply app/views/pages/moving-guide/civilians.html.erb]: how about Defense Travel Management Office’s Pet FAQ

File: app/views/pages/moving-guide/civilians.html.erb (hunk 10 above)
Comment (aileendds) [reply app/views/pages/moving-guide/civilians.html.erb]: The content is nearly identical and probably should be identical. I'll make a partial

**Pull request merged**